src/wallet/coinselection.h

```diff
@@ -87,7 +87,7 @@ class COutput
     int nDepth;
 
     /** Pre-computed estimated size of this output as a fully-signed input in a transaction. Can be -1 if it could not be calculated */
-    int nInputBytes;
+    int m_input_bytes;
```

Comment:
6d54822cbcd466df7f819430e3be87e480e41e77: did you mean to do this in the previous commit (that would make this commit a bit smaller too)?

Reply:
Done